registry/handlers/api_test.go

```diff
@@ -2242,7 +2268,12 @@ func finishUpload(t *testing.T, ub *v2.URLBuilder, name reference.Named, uploadU
 	return resp.Header.Get("Location")
 }
 
-func doPushChunk(t *testing.T, uploadURLBase string, body io.Reader) (*http.Response, digest.Digest, error) {
+type chunkOption struct {
+	// Content-Range
+	cr string
```

Comment:
Looks like the `cr` property is used in a couple of places (and not only close to where it's defined here). Perhaps make the property self-describing so that the comment is not needed to understand its purpose.

```suggestion
	contentRange string
```

A comment could still be added to provide more details if you think that's useful, e.g.:

```suggestion
	// Content-Range header to set when pushing chunks
	contentRange string
```

However, also see my other comment below

Reply:
updated, thanks.